devicon.json

```
@@ -10103,6 +10103,30 @@
             }
         ]
     },
+    {
+        "name": "veevalidate",
+        "altnames": [],
+        "tags": [
+            "vuejs",
+            "vuejs-library",
+            "package"
+        ],
+        "versions": {
+            "svg": [
+                "original"
+            ],
+            "font": [
+                "original"
+            ]
+        },
+        "color": "#05c770",
```

This color is different from the one used in the svg
```suggestion
        "color": "#04c971",
```

---

Apologies, my mistake. Color has been updated.

---

No worries! 😁